src/components/asset-list/RecyclerAssetList2/WrappedCollectiblesHeader.tsx

```diff
@@ -1,9 +1,10 @@
-import React from 'react';
+import { ListHeaderMenu } from '@/components/list/ListHeaderMenu';
 import { Box, Inline, Text } from '@/design-system';
+import { NftCollectionSortCriterion, SortDirection } from '@/graphql/__generated__/arc';
+import { NftSort, useNftSort } from '@/hooks/useNFTsSortBy';
 import * as i18n from '@/languages';
-import { ListHeaderMenu } from '@/components/list/ListHeaderMenu';
-import { NftCollectionSortCriterion } from '@/graphql/__generated__/arc';
-import useNftSort from '@/hooks/useNFTsSortBy';
+import { colors } from '@/styles';
+import React from 'react';
```

Comment:
please turn off the import sorter extension. Makes it really hard to digest what actually changes here.